exercises/binary-search/instructions.md

```diff
@@ -0,0 +1,24 @@
+# Instructions
+
+Your task is to implement a binary search algorithm.
+
+A binary search algorithm works by dividing the sorted list in half and comparing the middle element with the item we're looking for.
+If the middle element is our item, then we're done.
+If not, then there are two options.
+Either the middle element is greater than our item, or it is less than our item.
+
+If the middle element is greater than our item, we can eliminate that number and all the numbers after it.
+If the middle element is less than our item, we can eliminate that number and all the numbers before it.
+Then we can repeat the process on the other half of the list.
```

Comment:
Maybe stress that we're working with the remaining values (which juxtaposes with eliminating the numbers)

```suggestion
Then we can repeat the process on the remaining, other half of the list.
```